mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping

```diff
@@ -1,2 +1,36 @@
 CLASS net/minecraft/unmapped/C_jjmsgutv net/minecraft/world/gen/feature/ConfiguredStructureFeatures
+	FIELD f_aaxxefnw MONUMENT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_anbjnztl JUNGLE_PYRAMID Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_apqafzvk DESERT_PYRAMID Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_bwyhbopf VILLAGE_TAIGA Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_dgbzmuae RUINED_PORTAL_MOUNTAIN Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_dygfetkd RUINED_PORTAL_JUNGLE Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_erwqosvs VILLAGE_SNOWY Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_exalyvfz BURRIED_TREASURE Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_fdubauqf SHIPWRECK_BEACHED Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_hmfiqbze MINESHAFT_MESA Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_hvehfxqu RUINED_PORTAL_NETHER Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_kbqsvqwp RUINED_PORTAL_DESERT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_myulukez IGLOO Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_njofseod NETHER_FOSSIL Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_nvvqhogx VILLAGE_PLAINS Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_ratyniga OCEAN_RUIN_COLD Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_rbmpllyo MANSION Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_scdvemqm RUINED_PORTAL Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_twsuifmm MINESHAFT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_tzercjrg SWAMP_HUT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_usigcrcw VILLAGE_DESERT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_utiflovj RUINED_PORTAL_SWAMP Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_uuiskzrz FORTRESS Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_vfzsuxhc OCEAN_RUIN_WARM Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_wecpkfyo STRONGHOLD Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_xblgtolq END_CITY Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_xzkhxtmv VILLAGE_SAVANNA Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_ygvamzbm PILLAGER_OUTPOST Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_yrvepueb BASTION_REMNANT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_yzkyruhi RUINED_PORTAL_OCEAN Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_zpuibhgo SHIPWRECK Lnet/minecraft/unmapped/C_cjzoxshv;
```

Comment:
Aren't these automatically named by stitch?

Reply:
Ill look into what stitch does, might not catch this